### ISSUE TITLE
Second gamepad overlay won't work

### ISSUE THREAD

**Issue by ve1la** [univrsal/input-overlay#367]
**Describe the bug**
I can setup only one input overlay. If I try to add a second one to show 2 controllers at the same time, one of the two won't work. 
If I set both input overlays to the same controller, they'll work. 

**To Reproduce**
Connect 2 controllers and try to setup each to its own overlay, one of the two controllers will not work (if you press buttons, the overlay won't show anything being pressed). It seems to be random which controller will be ignored by input overlay.
Both overlay will work correctly if set to the other controller.

**Expected behavior**
To be able to show as many different controllers as like, each in its own input overlay.

**Screenshots**

**Additional information:**
- 5.0.4
- https://obsproject.com/logs/Avjfrosa0yKIoBt8

**Additional context**
I used a Dualshock 4 (xbox 360 through DS4Windows) and a xbox 360 controller. 
In-game both my controllers work fine.
I also tried a fresh obs install, same issue.

**Comment by davirpp** [comment]
Some issue here, but with me, is the "player 2" that doesn't work. The plugin recognizes it, but no input is shown.

**Comment by univrsal** [comment]
Yeah there's an issue when using multiple gamepads. The issue is that I only have one so it makes it a bit hard to track down the issue.

**Comment by ve1la** [comment]
Hello, this issue can be closed now, I tried again today and I can use 2 controllers without issues!
I have another (small) issue but I'll make a different thread for it
